test/compare_seed_catalog.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+import argparse
+import sys
+import os
+
+def run_brogue_tests(seed_catalog_file, extra_args, max_level):
+    
+    output_seed_catalog_file = f'output_seed_catalog_{max_level}.txt'
+    seed_catalog_cmd_args = f'--print-seed-catalog 1 100 {max_level}'
+
+    # Generate the first 100 seeds
+    if extra_args:
+        brogue_command = f'./brogue {extra_args} {seed_catalog_cmd_args} > {output_seed_catalog_file}'
+    else:
+        brogue_command = f'./brogue {seed_catalog_cmd_args} > {output_seed_catalog_file}'
+   
+    print(f"Running {brogue_command}...")
+    brogue_result = subprocess.run(brogue_command, shell=True, capture_output=True, text=True)
+
+    # Run diff to compare the seed catalog files
+
+    print (f"Comparing {seed_catalog_file} and {output_seed_catalog_file}...")
+
+    diff_command = f"bash -c 'diff <(sed \"1,4d\" {seed_catalog_file}) <(sed \"1,4d\" {output_seed_catalog_file})'"
```

Comment:
note: Another option is using CSV output which iirc has no preamble

Reply:
Noted, I prefer the text version for readability, but if it turns out to be too fragile we could use CSV